src/libs/OptionsListUtils.ts

```diff
@@ -1009,6 +1013,23 @@ function orderReportOptionsWithSearch(
     );
 }
 
+function orderWorkspaceOptions(options: ReportUtils.OptionData[]): ReportUtils.OptionData[] {
+    return lodashOrderBy(
```

Comment:
I can't remember the exact docs link, but isn't our preference to use built-in methods wherever possible? Can we use `options.sort` ?

Reply:
Done